leopard.yaml

```diff
@@ -0,0 +1,34 @@
ce of prostate cancer cells. This recurrence serves as a prognostic indicator for progression to clinical metastases and eventual prostate cancer-related mortality ([C. L. Amling, 2014;](https://pubmed.ncbi.nlm.nih.gov/10840432/) [S. J. Freedland et al., 2005;](https://jamanetwork.com/journals/jama/fullarticle/201291) [M. Han et al., 2001;](https://www.sciencedirect.com/science/article/abs/pii/S0094014305701634?via%3Dihub) [T. Van den Broeck et al., 2001](http://prognostic%20value%20of%20biochemical%20recurrence%20following%20treatment%20with%20curative%20intent%20for%20prostate%20cancer:%20A%20Systematic%20Review/)). Current clinical practices gauge the risk of biochemical recurrence by considering the International Society of Urological Pathology (ISUP) grade, PSA value at diagnosis, and TNM staging criteria ([J. I. Epstein et al., 2016](https://journals.lww.com/ajsp/abstract/2016/02000/the_2014_international_society_of_urological.10.aspx)). A recent European consensus guideline suggests categorizing patients into low-risk, intermediate-risk, and high-risk groups based on these factors ([N. Mottet et al., 2021](https://www.sciencedirect.com/science/article/abs/pii/S0302283820307697?via%3Dihub)). Notably, a high ISUP grade independently assigns a patient to the intermediate (grade 2/3) or high-risk group (grade 4/5). The Gleason growth patterns, representing morphological patterns of prostate cancer, are used to categorize cancerous tissue into ISUP grade groups ([J. I. Epstein, 2010;](https://www.auajournals.org/doi/10.1016/j.juro.2009.10.046) [P. M. Pierorazio et al., 2013;](https://bjui-journals.onlinelibrary.wiley.com/doi/10.1111/j.1464-410X.2012.11611.x) [G. J. L. H. van Leenders et al., 2020;](https://journals.lww.com/ajsp/fulltext/2020/08000/the_2019_international_society_of_urological.1.aspx) [J. I. Epstein et al., 2016](https://www.sciencedirect.com/science/article/abs/pii/S0302283815005576?via%3Dihub)). However, the ISUP grade has limitations, such as grading disagreement among pathologists ([J. I. Epstein et al., 2016](https://www.sciencedirect.com/science/article/abs/pii/S0302283815005576?via%3Dihub)) and coarse descriptors of tissue morphology. Recently, deep learning was shown ([H. Pinckaers et al., 2022](https://www.nature.com/articles/s43856-022-00126-3); [O. Eminaga et. al., 2024](https://www.nature.com/articles/s44303-023-00005-z?utm_source=rct_congratemailt&utm_medium=email&utm_campaign=oa_20240306&utm_content=10.1038/s44303-023-00005-z)) to be able to predict the biochemical recurrence of prostate cancer.  Hypothesizing that deep learning could uncover finer morphological features' prognostic value, we are organizing the LEarning biOchemical Prostate cAncer Recurrence from histopathology sliDes (LEOPARD) challenge. The goal of this challenge is to yield top-performance deep learning solutions to predict the time to biochemical recurrence from H&E-stained histopathological tissue sections, i.e. based on morphological features.
```

Comment:
For this url (https://www.nature.com/articles/s44303-023-00005-z?utm_source=rct_congratemailt&utm_medium=email&utm_campaign=oa_20240306&utm_content=10.1038/s44303-023-00005-z) can you confirm this is your tracking campaign appended? If it's not, using https://www.nature.com/articles/s44303-023-00005-z would be better here.

Reply:
Hi @KhrystynaFaryna, yes please push that change and also just confirm this URL works from your YAML - [T. Van den Broeck et al., 2001](http://prognostic%20value%20of%20biochemical%20recurrence%20following%20treatment%20with%20curative%20intent%20for%20prostate%20cancer:%20A%20Systematic%20Review/)) I cannot seem to get it to work.

Once those are done I will merge.

Reply:
I updated the .yaml